client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -40,8 +40,8 @@ class SharingServiceDescription extends Component {
 			instagram_business: function () {
 				if ( this.props.numberOfConnections > 0 ) {
 					return this.props.translate(
-						'Sharing photos and video reels to your Instagram account.',
-						'Sharing photos and video reels to your Instagram accounts.',
+						'Sharing photos to your Instagram account.',
```

Comment:
These should be 'Share photos to your Instagram account.

Reply:
I think no, [those are here](https://github.com/Automattic/wp-calypso/pull/77929/files/9cd2f3a3ad6965eae4648acbd3f8ab887fcbdd61#diff-039afe5cb9721f185926d23e7b755e388fcbe7f24f624e5f897cd1645b935d68L56). We say "Share photos" while there is no connected account, and say "Sharing photos.." when they have at least one connection. At least that's how the other services work

Reply:
Thanks for explaining it, Geri! You're right, that's how the other services work as well.